src/platforms/hydrabus/platform.c

```diff
@@ -36,6 +37,11 @@
 
 jmp_buf fatal_error_jmpbuf;
 
+int platform_hwversion(void)
```

Comment:
Is there a particular reason for removing this from the platform header? This will impair the optimiser's ability to optimise this function out to just a value in all call sites